.github/workflows/auto-triage-issues

```diff
@@ -0,0 +1,20 @@
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  auto_label:
+    runs-on: ubuntu-latest
+    name: Automatic Github Issue Labeller
+    steps:
+    - name: Label Step
```

Comment:
It can be renamed to something which is more intuitive, such as "Auto labeling issues"

Reply:
Done